### DISCUSSION TITLE
Installed in docker. What to do with "As a final step, you must add a scheduled task .."?

### DISCUSSION THREAD

**Discussion by ruslan-mogilevskiy** [SolidInvoice/SolidInvoice#1070] [Q&A]
Literally, the subject. I've installed Solid Invoice in docker and as a last step it shown this:
![2024-05-24_14h08_28](https://github.com/SolidInvoice/SolidInvoice/assets/69425331/918b36cb-5ac9-4e7e-adf0-901e48df8acb)

What should I do with this taking into account the docker hosting?

**Comment by pierredup** [comment]
Since version 2.3+, this step is not necessary, since it's now handled automatically.
